profiles/ros/base

```diff
@@ -11,13 +11,15 @@
 # Include base and networking abstractions needed for ros
 #include <abstractions/base>
 #include <abstractions/nameservice>
-#include <abstractions/user-tmp>
+
+# CHECKME: Is the rest of this no longer seems necessary?
+# #include <abstractions/user-tmp>
 
 # Allow unconfined processes to send us signals by default
-signal (receive) peer=unconfined,
+# signal (receive) peer=unconfined,
 
 # Allow us to receive signals from ros tools
-signal receive peer=ros/roslaunch,
+# signal receive peer=ros/roslaunch,
```

Comment:
We should be able to remove the roslaunch example https://github.com/ros-infrastructure/apparmor_profiles/blob/4418fada82e47fca862211ebf0a4065b93de0dcc/examples/ros/roslaunch_talker_listener

Or to update it with a profile for `ros2 launch demo_nodes_cpp talker_listener.launch.py`

Reply:
Yea, I haven't touched the launch example files as I've yet to profile the permission requirements for ROS2's launch process. I'm also not sure of the AppArmor profile language as changed at all here, perhaps there alternate methods of scoping signal prevleage by now.